examples/allocator-service/allocator-service.yaml

```diff
@@ -0,0 +1,60 @@
+# Define a Service for the fleet-allocator
+apiVersion: v1
+kind: Service
+metadata:
+  name: fleet-allocator-backend
+  annotations:
+    service.alpha.kubernetes.io/app-protocols: '{"https":"HTTPS"}'
+  labels:
+    app: fleet-allocator
+spec:
+  type: NodePort
+  selector:
+    app: fleet-allocator
+  ports:
+  - port: 8000
+    protocol: TCP
+    name: https
+    targetPort: fleet-allocator  # retrieve port from deployment config
+
+---
+# Deploy a pod to run the fleet-allocator code
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fleet-allocator
+  namespace: default
+  labels:
+    app: fleet-allocator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fleet-allocator
+  template:
+    metadata:
+      labels:
+        app: fleet-allocator
+    spec:
+      serviceAccount: fleet-allocator
+      volumes:
+      - name: secret-volume
+        secret:
+          secretName: allocatorw3secret
+      containers:
+      - name: fleet-allocator
+        image: index.docker.io/trota/allocator-service:simple-udp-0.5.0-03f4866-5
```

Comment:
Yes, providing instructions on how to build it would definitely be better.  I imagine most of the audience will be comfortable hosting their own image in their own repository.
I'll make these changes.